### ISSUE TITLE
NTR: 'Abnormality of fingers 2-5'

### ISSUE THREAD

**Issue by nicolevasilevsky** [obophenotype/human-phenotype-ontology#4146]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Abnormality of fingers 2-5

### 2. Synonyms
Abnormality of fingers but not thumb

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)
An anomaly of fingers 2-5 (index, middle, ring and pinky finger).

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp))
'Abnormality of finger'

**Comment by nicolevasilevsky** [comment]
For the Kid's First data portal. The description is 'Abnormal Fingers but not Thumb'. I could just annotate abnormality of the 2nd finger, etc.

**Comment by pnrobinson** [comment]
It seems a lot of the Kids First data portal items are general descriptions and not great HPO terms.
In this case, I would recommend abnormality of finger and NOT abn of thumb or something like that.

**Comment by allisonheath** [comment]
Thanks @pnrobinson that makes sense - @nicolevasilevsky I think in cases like that splitting into multiple terms like that would help compile how we should support these. Just looking at it, seems like you would do positively observed abnormality for 2nd, 3rd, 4th, 5th and negatively observed for thumb:

<img width="288" alt="screen shot 2018-10-05 at 2 39 49 pm" src="https://user-images.githubusercontent.com/2213854/46553829-8bc9fa00-c8ac-11e8-9978-93c4f7e8c765.png">

**Comment by nicolevasilevsky** [comment]
@allisonheath sounds good!
